pkg/controller/cloud/node_controller.go

```diff
@@ -122,9 +112,6 @@ func (cnc *CloudNodeController) Run(stopCh <-chan struct{}) {
 
```

Comment:
For line 114: Some comments may need to be updated in this file, since now this controller doesn't delete nodes anymore.